components/pkg-export-docker/src/cli.rs

```diff
@@ -283,6 +283,29 @@ impl<'a, 'b> Cli<'a, 'b> {
 
         Cli { app: app }
     }
+
+    pub fn add_memory_arg(self) -> Self {
+        let app = self.app.arg(
+            Arg::with_name("MEMORY_LIMIT")
+                .value_name("MEMORY_LIMIT")
+                .long("memory")
+                .short("m")
+                .help("Memory limit allocated to build"),
```

Comment:
according to the online docker run docs:

```
Memory limit (format: <number>[<unit>]). Number is a positive integer. Unit can be one of b, k, m, or g. Minimum is 4M
```

However this is fairly incomplete since I know `GB` works too. Their CLI docs dont specify the format.